scripts/deploy.py

```diff
@@ -18,6 +18,20 @@
 
 
 if __name__ == '__main__':
+
+    print("Decrypting secrets")
+
+    subprocess.check_call(
+        "openssl aes-256-cbc -K $encrypted_39cb4cc39a80_key "
+        "-iv $encrypted_39cb4cc39a80_iv -in secrets.tar.enc "
+        "-out secrets.tar -d",
+        shell=True
```

Comment:
I am too, but I thought it worth making a special case for this because it's basically a magic string from Travis that we don't really know  or want to know the specifics of, and it's a relatively high cost to convert it to non-shell. Additionally there is zero programmatic input in building the string.